account.py

```diff
@@ -712,7 +712,7 @@ def get_extra_lines(self, lines, account, party):
                     origins[line.origin] = Decimal('0.0')
                 origins[line.origin] += abs(line_amount)
 
-        if not account or not party:
+        if not account:
```

Comment:
I think it's not correct. The extra line is the compensation line, the result of the Debit-Credit of the move lines selected.
Without the extra line, there is no compensation.
Need to create this extra line, with the account selected, but the party won't be required.
Not sure the problem was here.